peps/pep-0768.rst

```diff
@@ -322,6 +329,37 @@ feature. This way, tools can offer a useful error message explaining why they
 won't work, instead of believing that they have attached and then never having
 their script run.
 
+Multi-threading Considerations
+------------------------------
+
+Debugging code injected through this interface executes opportunistically in
+whichever thread first encounters a safe evaluation point after the request is
+made. This behavior mirrors how Python handles signals, providing a reliable
```

Comment:
Hm? Signal handlers always run in the main thread, unlike the debugger script...

Reply:
And, wait, this just seems wrong to me: doesn't debugging code injected through this interface execute _specifically_ in the thread whose thread state the debugger script was set in, not in an arbitrary thread? This just seems wrong to me.

Reply:
Sorry I wanted to say that the debugger wants to inject it into an arbitrary thread. Will rewrite this part